CHANGELOG.rst

```diff
@@ -109,6 +112,10 @@ Fixed
 
   Contributed by Tatsuma Matsuki (@mtatsuma)
 
+* Fix dependency conflicts by updating ``requests`` to latest version (2.23.0). #4869
+* Fix dependency conflicts by updating ``gitpython`` to version 2.1.15 #4869
+* Fix travis issue that prevented root from writing virtualenv lock files under /home/travis. #4869
```

Comment:
@blag We usually don't include Travis, CI/CD, Makefile or any other build-related changes in `st2` Changelog.
What matters is `stackstorm/st2` core platform or whether its behavior changed or not.
And so in this situation, only `pip` changes are relevant for the changelog.

@cognifloyd Ideally if we'll have 1 line in Changelog for that bugfix.

Sorry for confusion.

Reply:
I initially left out the build changes and only mentioned the requirements updates, so what you say makes sense. I changed to meet one Senior Maintainers request, and now I have another Senior Maintainer requesting the opposite. So, @blag, are you ok with @armab's request, with going back to one requirements entry?

Reply:
Yes, I'm fine with that.

Reply:
done.